pkg/grpc/grpc.go

```diff
@@ -0,0 +1,232 @@
+package grpc
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	base "cosmossdk.io/api/cosmos/base/tendermint/v1beta1"
+
+	"github.com/archway-network/validator-exporter/pkg/config"
+	"github.com/archway-network/validator-exporter/pkg/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/bech32"
+	"github.com/cosmos/cosmos-sdk/types/module/testutil"
+	"github.com/cosmos/cosmos-sdk/types/query"
+	slashing "github.com/cosmos/cosmos-sdk/x/slashing/types"
+	staking "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"google.golang.org/grpc"
+
+	log "github.com/archway-network/validator-exporter/pkg/logger"
+)
+
+const valConsStr = "valcons"
+
+var errEndpoint = errors.New("grpc endpoint error")
```

Comment:
I'd name it based on the package, so just `grpc error` , the extra information then will be appended